src/renderer/families/solana/SendRecipientFields.js

```diff
@@ -4,7 +4,6 @@ import { Trans, withTranslation } from "react-i18next";
 import MemoValueField from "./MemoValueField";
 import Box from "~/renderer/components/Box";
 import Label from "~/renderer/components/Label";
-import LabelInfoTooltip from "~/renderer/components/LabelInfoTooltip";
```

Comment:
Needed to fix a lint issue

> /actions-runner/_work/ledger-live-desktop/ledger-live-desktop/src/renderer/families/solana/SendRecipientFields.js
7:8 error 'LabelInfoTooltip' is defined but never used no-unused-vars

cf. https://github.com/LedgerHQ/ledger-live-desktop/pull/4597#issuecomment-1004012161